src/shadowbox/server/manager_service.ts

```diff
@@ -147,6 +149,37 @@ export class ShadowsocksManagerService {
     next();
   }
 
+  // Changes the server's hostname.  Hostname must be a valid domain or IP address
+  public changeHostname(req: RequestType, res: ResponseType, next: restify.Next): void {
```

Comment:
nit: for consistency, should this be `setHostnameForAccessKeys`?

Reply:
Waiting to resolve until we decide which access keys are affected

Reply:
Done